manifests/compose/dev/compose.yaml

```diff
@@ -1,13 +1,18 @@
 name: model-dev
+include:
+  - path:
+      - ../monitoring/compose.yaml
+      - ./overrides.yaml
+
 services:
-  kepler:
+  kepler-models:
     image: quay.io/sustainable_computing_io/kepler:latest
     ports:
-      - 9100:9100
+      - 19100:9100
```

Comment:
@sthaha Any specific reason why are we using `1<port_number>` as host port?

Reply:
Yes, this enables running model-server development compose and kepler dev composes (metal specifically) simultaneously

Reply:
But we do run kepler metal as part of model-server compose?